lib/benchee/configuration.ex

```diff
@@ -353,7 +353,16 @@ defmodule Benchee.Configuration do
     end
   end
 
-  defp force_string_input_keys(config = %{inputs: inputs}) do
+  defp force_string_input_keys(config = %{inputs: inputs}) when is_list(inputs) do
+    standardized_inputs =
+      Enum.map(inputs, fn {name, value} ->
+        {to_string(name), value}
+      end)
+
+    %{config | inputs: standardized_inputs}
```

Comment:
I wonder if it wouldn't make sense to coerce both lists and maps in here to a single representation? In particular always to make it a list of tuples. That way further parts of the code don't need to be concerned with the differences in the user interface.

Reply:
Good point - updated!